templates/advancement/trait-config-choices.hbs

```diff
@@ -0,0 +1,31 @@
+<fieldset class="selected-trait">
+    <legend>
+        {{ localize "DND5E.ADVANCEMENT.Trait.FIELDS.choices.label" }}
+        <a data-action="addChoice" data-tooltip="DND5E.ADVANCEMENT.Trait.Action.AddChoice"
+           aria-label="{{ localize 'DND5E.ADVANCEMENT.Trait.Action.AddChoice' }}">
+            <i class="fas fa-plus" inert></i>
+        </a>
```

Comment:
Should probably be a `button.control-button`. As far as I understand WCAG guidelines anyway, anchor elements that behave like buttons (i.e. don't actually link to anything and instead perform some action on click) should just be buttons.